src/main/java/com/teamzunior/stringcalculator/calculator/Calculator.java

```diff
@@ -0,0 +1,49 @@
+package com.teamzunior.stringcalculator.calculator;
+
+import java.math.BigDecimal;
+
+/**
+ * Created by qkrtjdehd123 on 2020-01-08
+ */
+public final class Calculator {
+
+    /** 값을 쌓는 단지. **/
+    private final ValuePot pot = ValuePot.createPot();
+
+    private Calculator(){ }
+
+    public static Calculator createOfCalculator() {
```

Comment:
사용시 호출을 `Calculator.createOfCalculator` 라고 하니까  Calculator를 굳이 두번 써줄 필요는 없을 것 같아요! 
```suggestion
    public static Calculator newInstance() {
```
```suggestion
    public static Calculator create() {
```

정적 팩토리 메소드 패턴 👍

Reply:
`create()` 표현이 더 간결하겠군요 👍